core/src/main/scala/org/apache/spark/sql/rapids/tool/AppBase.scala

```diff
@@ -293,9 +293,10 @@ abstract class AppBase(
       val readSchema = ReadParser.formatSchemaStr(meta.getOrElse("ReadSchema", ""))
       val scanNode = allNodes.filter(node => {
         // Get ReadSchema of each Node and sanitize it for comparison
+        // The "NativeScan" name is from Velox engine
```

Comment:
This is very easy to be changed. Better to think of a better way for this. I realize this can be as a follow-up.